sds/src/scanner/mod.rs

```diff
@@ -88,6 +91,7 @@ impl Scanner {
             rules: rules.clone(),
             scoped_ruleset,
             cache_pool: CachePool::new(rules),
+            metrics: Metrics::new(&scanner_labels),
```

Comment:
[`logs.sds.false_positives.v0` is tagged by `scrubbing_rule_name`](https://app.datadoghq.com/dashboard/veg-ziv-2ea/logs-sds-overview?fromUser=false&fullscreen_end_ts=1714983264788&fullscreen_paused=false&fullscreen_refresh_mode=sliding&fullscreen_section=edit&fullscreen_start_ts=1714968864788&fullscreen_widget=5222798691058174&refresh_mode=sliding&view=spans&from_ts=1714968558197&to_ts=1714982958197&live=true). With the current implementation, we will lose this tag. Would it be possible to keep this tag?

Reply:
I re-implemented this to include the tags of the rule: [Create metric for each compiled rule](https://github.com/DataDog/dd-sensitive-data-scanner/pull/40/commits/7f28c8b3ee2bc2dfa8939a69cb4cb5f0ab45cd09)